python/tests/conftest.py

```diff
@@ -37,3 +37,14 @@ def get_oai_config():
         api_key, org_id = sk.openai_settings_from_dot_env()
 
     return api_key, org_id
+
+
+@pytest.fixture(scope="session")
+def get_gp_config():
```

Comment:
For now, do you mind marking the google palm integration tests as skipped? 

pytestmark = pytest.mark.skipif(
    "Python_Integration_Tests" in os.environ:
    reason="Google Palm integration tests are only set up to run locally",
)

Reply:
Ok, it's fixed